service/spanner/spanner.go

```diff
@@ -46,21 +44,15 @@ func ParseDDL(updateDB bool) error {
 					models.OriginalColResponse[column] = originalColumn
 				}
 			}
-			if currentTable != tableName {
-				if currentTable != "" {
-					models.TableColumnMap[currentTable] = tmpCols
-				}
+			_, found := models.TableColumnMap[tableName]
+			if !found {
 				models.TableDDL[tableName] = make(map[string]string)
 				models.TableColumnMap[tableName] = []string{}
-				tmpCols = []string{}
-				currentTable = tableName
 			}
-			tmpCols = append(tmpCols, column)
+			models.TableColumnMap[tableName] = append(models.TableColumnMap[tableName], column)
 			models.TableDDL[tableName][column] = dataType
 		}
-		models.TableColumnMap[currentTable] = tmpCols
 	}
-
```

Comment:
Why did you change it? any reason?

Reply:
yes because it was working fine if we have all the columns info of a table stored continuously but when we use Spanner emulator it doesn't maintain order so need to change it so that it can work if there is an order or not.